dojo/tools/nmap/parser.py

```diff
@@ -126,11 +128,15 @@ def get_findings(self, file, test):
                         severity=severity,
                         mitigation="N/A",
                         impact="No impact provided",
+                        vuln_id_from_tool=script_id,
```

Comment:
This could be in a separate request, as it's a clear improvement over the existing code that doesn't depend on the addition of the `Problem` functionality.

Reply:
I created a new PR #11454